client/post-editor/edit-post-status/index.jsx

```diff
@@ -52,48 +57,50 @@ export default React.createClass( {
 			stickyEventLabel = 'Off';
 		}
 
-		stats.recordStat( stickyStat );
-		stats.recordEvent( 'Changed Sticky Setting', stickyEventLabel );
+		recordStat( stickyStat );
+		recordEvent( 'Changed Sticky Setting', stickyEventLabel );
 
-		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
-		actions.edit( { sticky: ! this.props.post.sticky } );
-	},
+		this.props.editPost( this.props.siteId, this.props.postId,
```

Comment:
Yes! sometimes it's hard to find the right way to write long expressions, I'll take yours